edx_rest_framework_extensions/tests/test_utils.py

```diff
@@ -38,10 +72,70 @@ def setUp(self):
         self.jwt = generate_jwt_token(self.payload)
 
     def test_decode_success(self):
-        self.assertDictEqual(jwt_decode_handler(self.jwt), self.payload)
+        """
+        Confirms that the format of the valid response from the token decoder matches the payload
+        """
+        self.assertDictEqual(utils.jwt_decode_handler(self.jwt), self.payload)
+
+    def test_decode_valid_token_multiple_valid_issuers(self):
+        """
+        Validates that a valid token is properly decoded given a list of multiple valid issuers
+        """
 
-    def test_decode_error(self):
+        # Verify that each valid issuer is properly matched against the valid issuers list
+        # and used to decode the token that was generated using said valid issuer data
+        settings.JWT_AUTH['JWT_ISSUERS'] = VALID_ISSUERS
```

Comment:
Actually, you are already defining values in settings. Use those, and get rid of `VALID_ISSUERS`.